### PR TITLE
Remove support for epoll level-triggered

### PR DESCRIPTION
**Pull request by normanmaurer**
Motivation:

Netty uses epoll edge-triggered by default forever and there is really not reason why someone should use level-triggered (its considered a implementation detail).

Modifications:

- Remove code that was related to level-triggered mode
- Adjust testclass names

Result:

Fixes https://github.com/netty/netty/issues/9349

### CODE REVIEW

**Comment by normanmaurer** [comment]
This currently depend on https://github.com/netty/netty/pull/9825.

**Comment by normanmaurer** [comment]
@netty-bot test this please

File: transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollServerChannel.java [hunk 1]
```diff
@@ -102,7 +102,6 @@ void epollInReady() {
                 return;
             }
             final EpollRecvByteAllocatorHandle allocHandle = recvBufAllocHandle();
-            allocHandle.edgeTriggered(isFlagSet(Native.EPOLLET));
```
Comment (johnou) [inline transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollServerChannel.java]: is the EPOLLET still used in other places?

Comment (normanmaurer) [reply transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollServerChannel.java]: yep it is...

**Comment by normanmaurer** [comment]
@njhill yes lets see if it worth it for 4.1

**Pull request merged**